brats/metrics.py

```diff
@@ -0,0 +1,32 @@
+import torch
+
+ZERO_AND_ONE = torch.tensor([0., 1.])
+CHANNELS_DIM = 1
+ONE_CLASS = 1
+
+
+def dice_score_one_class(prediction: torch.Tensor, target: torch.Tensor,
```

Comment:
There is a large amount of code overlapping with dice loss. Is it possible to extract it somehow and use in both loss and metric?

Reply:
Addressed. Extracted intersection and union calculations to separate functions and utilized dice **score** calculation in the dice **loss**